lib/engine/game/g_1870/step/connection_token.rb

```diff
@@ -39,7 +41,11 @@ def process_choose(action)
             destination.remove_assignment!(entity)
 
             if action.choice == 'Map'
-              destination.tile.cities.first.place_token(entity, token, free: true, extra_slot: true)
+              if @game.station_wars? && STATION_WARS_CORPS.include?(entity.id)
+                destination.tile.cities.first.place_token(entity, token, free: true, cheater: true)
+              else
+                destination.tile.cities.first.place_token(entity, token, free: true, extra_slot: true)
+              end
```

Comment:
I find it more readable to have only one call with a lot of arguments, with the code inside of the conditional only saying what's different with the variant or not, something like this (I don't think `token_args` is a great name, please change it if you have a better idea):

```suggestion
              token_args =
                if @game.station_wars? && STATION_WARS_CORPS.include?(entity.id)
                  { cheater: true }
                else
                  { extra_slot: true }
                end
              destination.tile.cities.first.place_token(entity, token, free: true, **token_args)
```

Reply:
cool suggestion! I went with token_placement_arg. submitted.